appserver/monitoring-console/webapp/src/main/java/fish/payara/monitoring/web/ApiResponses.java

```diff
@@ -0,0 +1,326 @@
+ * The contents of this file are subject to the terms of either the GNU
+ * General Public License Version 2 only ("GPL") or the Common Development
+ * and Distribution License("CDDL") (collectively, the "License").  You
+ * may not use this file except in compliance with the License.  You can
+ * obtain a copy of the License at
+ * https://github.com/payara/Payara/blob/master/LICENSE.txt
+ * See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing the software, include this License Header Notice in each
+ * file and include the License file at glassfish/legal/LICENSE.txt.
+ *
+ * GPL Classpath Exception:
+ * The Payara Foundation designates this particular file as subject to the "Classpath"
+ * exception as provided by the Payara Foundation in the GPL Version 2 section of the License
+ * file that accompanied this code.
+ *
+ * Modifications:
+ * If applicable, add the following below the License Header, with the fields
+ * enclosed by brackets [] replaced by your own identifying information:
+ * "Portions Copyright [year] [name of copyright owner]"
+ *
+ * Contributor(s):
+ * If you wish your version of this file to be governed by only the CDDL or
+ * only the GPL Version 2, indicate your decision by adding "[Contributor]
+ * elects to include this software in this distribution under the [CDDL or GPL
+ * Version 2] license."  If you don't indicate a single choice of license, a
+ * recipient has the option to distribute your version of this file under
+ * either the CDDL, the GPL Version 2 or to extend the choice of license to
+ * its licensees as provided above.  However, if you add GPL Version 2 code
+ * and therefore, elected the GPL Version 2 license, then the option applies
+ * only if the new code is made subject to such option by the copyright
+ * holder.
+ */
+package fish.payara.monitoring.web;
+
+import static java.util.stream.Collectors.toList;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import fish.payara.monitoring.alert.Watch;
+import fish.payara.monitoring.alert.Alert;
+import fish.payara.monitoring.alert.Alert.Level;
+import fish.payara.monitoring.alert.AlertService.AlertStatistics;
+import fish.payara.monitoring.alert.Circumstance;
+import fish.payara.monitoring.alert.Condition;
+import fish.payara.monitoring.model.SeriesDataset;
+import fish.payara.monitoring.web.ApiRequests.SeriesQuery;
+import fish.payara.notification.requesttracing.RequestTrace;
+import fish.payara.nucleus.requesttracing.RequestTracingService;
+
+/**
+ * Types used in the web API to model mapped responses.
+ * 
+ * The purpose of these classes is to decouple the API from internal server classes so that renaming or restructuring of
+ * internal classes does not break the API.
+ * 
+ * @see ApiRequests
+ * 
+ * @author Jan Bernitt
+ */
+public final class ApiResponses {
```

Comment:
NB: I intentionally moved all the structs for the JSON API responses into one class because I want to look at them at the same time and I don't want to have numerous files open I have to navigate in-between when checking or changing the API.